samples/basic/blink_led/src/main.c

```diff
@@ -36,6 +36,9 @@
 #elif defined(CONFIG_BOARD_COLIBRI_IMX7D_M4)
 #define PWM_DRIVER	PWM_1_LABEL
 #define PWM_CHANNEL	0
+#elif defined(CONFIG_SOC_FAMILY_NRF)
```

Comment:
need to add the platforms to the whitelist in samples/basic/blink_led/sample.yaml

Reply:
ok

Reply:
ok